rust/datafusion/src/physical_plan/hash_join.rs

```diff
@@ -48,23 +54,15 @@ use super::{
 use crate::error::{DataFusionError, Result};
 
 use super::{ExecutionPlan, Partitioning, RecordBatchStream, SendableRecordBatchStream};
+use crate::physical_plan::coalesce_batches::concat_batches;
```

Comment:
Is there a place to put this?

Reply:
It should probably go into the arrow crate since it isn't specific to DataFusion.